### PR TITLE
Bxmsdoc 2396 master

### PR DESCRIPTION
**Pull request by gadeyredhat**
Links:

[JIRA](https://issues.jboss.org/browse/BXMSDOC-2589)
[Doc preview for PAM 7.0](http://file.pnq.redhat.com/~gadey/BXMSDOC-2396/)

### CODE REVIEW

File: docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-pages-con.adoc [hunk 1]
```diff
@@ -0,0 +1,15 @@
+[id='_building_custom_dashboard_widgets_pages_con']
+= Pages
+
+A page, also known as a perspective in technical or developer jargon, is a logical grouping of related panels, parts and components. A page consists of the following three types of components:
+
+* Core Components
+* Navigational Components
+* Reporting Components
+
+A page will have (one or more) core component(s) and may or may not have navigational and reporting components. The Workbench supports both developer created pages as well as end user created ones. The *Dashbuilder* (Content Management) tool is used to create the pages.
```
Comment (dgutierr) [inline docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-pages-con.adoc]: It is not mandatory to have a core component. So I'd say "A page might have zero or multiple instance of any of the component types available"

Comment (dgutierr) [inline docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-pages-con.adoc]: I wouldn't use the term "Dashbuilder" in general as this is a deprecated name. Better to refer to it as either the "Page authoring tooling" or the "Page builder".

File: docs/product-user-guide/src/main/asciidoc/data-sets-add-proc.adoc [hunk 2]
```diff
@@ -1,18 +1,21 @@
 [id='data_sets_add_proc']
 = Adding Data Sets
 
-You can define and add a new data set to fetch your data from an external storage system and begin using data sets in your Dashboard displayer.
+You can define and add a new data set to fetch your data from an external data source and begin using them in your Dashboard displayer.
```
Comment (dgutierr) [inline docs/product-user-guide/src/main/asciidoc/data-sets-add-proc.adoc]: The Displayer component is also a deprecated concept. We should refer to it as "reporting components".

File: docs/product-assembly_building-custom-dashboard-widgets/src/main/asciidoc/main.adoc [hunk 3]
```diff
@@ -0,0 +1,35 @@
+[id='assembly_building-custom-dashboard-widgets']
+
+include::product-shared-docs/document-attributes.adoc[]
+
+= Building custom dashboard widgets using the Dashbuilder (Content Management) tool
```
Comment (dgutierr) [inline docs/product-assembly_building-custom-dashboard-widgets/src/main/asciidoc/main.adoc]: The Dashbuilder term is deprecated. Please do not use it anywhere in the document. Use "Page authoring", "Page builder" or "Page editor" instead.

File: docs/product-assembly_building-custom-dashboard-widgets/src/main/asciidoc/main.adoc [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+// Context attribute is assembly specific and enables module reuse between assemblies.
+:imagesdir: topics/product-shared-docs/images
+:context: building-custom-dashboard-widgets
+include::product-shared-docs/author-group.adoc[]
+
+// Purpose statement for the assembly
+As a business analyst or business rules developer, you will use the page builder to design and define your page and determine what all information can be displayed through it. You can decide what all core, navigational and reporting components are a part of your page. You can also customize the properties of the components of the page as per your requirements. It is recommended that you define a data provider prior to creating a page.
```
Comment (dgutierr) [inline docs/product-assembly_building-custom-dashboard-widgets/src/main/asciidoc/main.adoc]: So far we provide only core, navigational and reporting components. This is likely to be extended in the near future. So I'd refer to them as "the page builder components", "component palette" or just "components" depending on the case.

File: docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-creating-pages-proc.adoc [hunk 5]
```diff
@@ -0,0 +1,12 @@
+[id='_building_custom_dashboard_widgets_creating_pages_proc']
+= Creating Pages
+
+Using the KIE Workbench, you can create a page which will contain core, navigational, and reporting components as per your requirements. After a page has been created and all the components in it defined, you can use the *Page Editor* tool of the *Dashbuilder* to save, delete, rename or copy pages as needed.
```
Comment (dgutierr) [inline docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-creating-pages-proc.adoc]: * "core, navigational, and reporting components" => "different components"?
* "Page Editor* tool of the Dashbuilder => "Page Authoring tool of the Workbench"?

File: docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-creating-pages-proc.adoc [hunk 6]
```diff
@@ -0,0 +1,12 @@
...
+
+The following procedure creates a blank page only and you will have to add the core, navigational, and reporting components as required manually.
```
Comment (dgutierr) [inline docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-creating-pages-proc.adoc]: the core, navigational, and reporting components => "the page components"?

File: docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-creating-pages-proc.adoc [hunk 7]
```diff
@@ -0,0 +1,12 @@
...
+
+.Procedure
+. Go to *Menu* -> *Design* -> *Pages*.
+. In the left pane, from the *Pages* tab, click *New*.
+  Alternatively, click on *New Page* from the *Content Manager* pane.
```
Comment (dgutierr) [inline docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-creating-pages-proc.adoc]: from the *Content Manager* pane.=> "from the *Page editor* pane"?

File: docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-pages-con.adoc [hunk 8]
```diff
@@ -0,0 +1,15 @@
+[id='_building_custom_dashboard_widgets_pages_con']
+= Pages
+
+A page, also known as a perspective in technical or developer jargon, is a logical grouping of related panels, parts and components. A page consists of the following three types of components:
```
Comment (dgutierr) [inline docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-pages-con.adoc]: A page consists of zero or more instances of the following components?

File: docs/product-user-guide/src/main/asciidoc/data-sets-con.adoc [hunk 9]
```diff
@@ -1,7 +1,15 @@
 [id='data_sets_con']
 = Data Sets
 
-The data set functionality in {CENTRAL} defines how to access and parse data.
-Data sets serve as a source of data that can be displayed by the Dashbuilder displayer.
-You can add the Dashbuilder displayers to a custom page in the *Plugin Management* page.
+A data set is a matrix of data which comprises of time stamps, texts and numbers. A data set can be stored in many ways, such as, in a database, in an excel file, in memory or in other ways. A data set definition, on the other hand, tells the Dashbuilder on how to access, read and parse a data set.
```
Comment (dgutierr) [inline docs/product-user-guide/src/main/asciidoc/data-sets-con.adoc]: * tells the Dashbuilder => "tells the Workbench" or alternatively "tells the UI"

File: docs/product-user-guide/src/main/asciidoc/data-sets-con.adoc [hunk 10]
```diff
@@ -1,7 +1,15 @@
...
+
+Note that the Dashbuilder will not store any data, it simply provides a way to define access to the data of a data set regardless of where that data is stored.
```
Comment (dgutierr) [inline docs/product-user-guide/src/main/asciidoc/data-sets-con.adoc]: * Note that the Dashbuilder = "Note that the Workbench" again never use the term Dashbuilder

File: docs/product-user-guide/src/main/asciidoc/data-sets-con.adoc [hunk 11]
```diff
@@ -1,7 +1,15 @@
...
+
+For example, let’s take an example of data stored in a database. A valid data set could be, for example, the entire database or a subset of the database (the result of an SQL query). In both cases, the database will return data in the form of a matrix of rows and columns. Now this data is used as input to the Dashbuilder Displayer which then displays this information in the form of charts or graphs.
```
Comment (dgutierr) [inline docs/product-user-guide/src/main/asciidoc/data-sets-con.adoc]: Dashbuilder Displayer => "reporting component". Please make sure Dashbuilder Displayer is replaced everywhere.

File: docs/product-assembly_building-custom-dashboard-widgets/src/main/asciidoc/dm/master-docinfo.xml [hunk 12]
```diff
@@ -0,0 +1,10 @@
+<productname>{PRODUCT}</productname>
+<productnumber>{PRODUCT_VERSION}</productnumber>
+<subtitle>
+	For {PRODUCT} {PRODUCT_VERSION}
+
+</subtitle>
+<abstract>
+	<para>This document describes how to design a decision service using guided decision tables in {PRODUCT} {PRODUCT_VERSION}.</para>
+</abstract>
```
Comment (vidya-iyengar) [inline docs/product-assembly_building-custom-dashboard-widgets/src/main/asciidoc/dm/master-docinfo.xml]: test comment

File: docs/product-assembly_building-custom-dashboard-widgets/src/main/asciidoc/main.adoc (hunk 3 above)
Comment (gadeyredhat) [reply docs/product-assembly_building-custom-dashboard-widgets/src/main/asciidoc/main.adoc]: Review accepted.

File: docs/product-assembly_building-custom-dashboard-widgets/src/main/asciidoc/main.adoc (hunk 4 above)
Comment (gadeyredhat) [reply docs/product-assembly_building-custom-dashboard-widgets/src/main/asciidoc/main.adoc]: Using just the words "components". Review accepted.

File: docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-creating-pages-proc.adoc (hunk 5 above)
Comment (gadeyredhat) [reply docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-creating-pages-proc.adoc]: Review accepted.

File: docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-creating-pages-proc.adoc (hunk 6 above)
Comment (gadeyredhat) [reply docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-creating-pages-proc.adoc]: Review accepted.

File: docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-creating-pages-proc.adoc (hunk 7 above)
Comment (gadeyredhat) [reply docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-creating-pages-proc.adoc]: Review accepted.

File: docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-pages-con.adoc (hunk 1 above)
Comment (gadeyredhat) [reply docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-pages-con.adoc]: Review accepted.

Comment (gadeyredhat) [reply docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-pages-con.adoc]: Review accepted.

File: docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-pages-con.adoc (hunk 8 above)
Comment (gadeyredhat) [reply docs/product-user-guide/src/main/asciidoc/building-custom-dashboard-widgets-pages-con.adoc]: Re-phrased the sentence. Review accepted.

File: docs/product-assembly_building-custom-dashboard-widgets/src/main/asciidoc/main.adoc (hunk 3 above)
Comment (gadeyredhat) [reply docs/product-assembly_building-custom-dashboard-widgets/src/main/asciidoc/main.adoc]: Using the "Page Authoring tool" term.

File: docs/product-user-guide/src/main/asciidoc/data-sets-add-proc.adoc (hunk 2 above)
Comment (gadeyredhat) [reply docs/product-user-guide/src/main/asciidoc/data-sets-add-proc.adoc]: Review accepted.

File: docs/product-user-guide/src/main/asciidoc/data-sets-con.adoc (hunk 9 above)
Comment (gadeyredhat) [reply docs/product-user-guide/src/main/asciidoc/data-sets-con.adoc]: Review accepted.

File: docs/product-user-guide/src/main/asciidoc/data-sets-con.adoc (hunk 10 above)
Comment (gadeyredhat) [reply docs/product-user-guide/src/main/asciidoc/data-sets-con.adoc]: Review accepted.

File: docs/product-user-guide/src/main/asciidoc/data-sets-con.adoc (hunk 11 above)
Comment (gadeyredhat) [reply docs/product-user-guide/src/main/asciidoc/data-sets-con.adoc]: Review accepted.

File: docs/product-user-guide/src/main/asciidoc/data-sets-edit-proc.adoc [hunk 13]
```diff
@@ -1,17 +1,17 @@
 [id='data_sets_edit_proc']
 = Editing Data Sets
 
-You can edit existing data sets as needed to ensure that fetched data in your Dashboard displayer is updated.
+You can edit existing data sets as per your requirement to ensure that the data fetched to the Dashboard displayer is up-to-date.
```
Comment (gadeyredhat) [inline docs/product-user-guide/src/main/asciidoc/data-sets-edit-proc.adoc]: Changed "Dashboard displayer" to "reporting components".

**Review by gadeyredhat — COMMENTED** [review]
Review comments accepted.

**Comment by gadeyredhat** [comment]
First SME review completed.

**Pull request closed without merging**